Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -418,16 +410,15 @@ public class KolodaView: UIView, DraggableCardDelegate {
             clear()
         }
         
-        if (countOfCards - (currentCardNumber + visibleCards.count) > 0) {
+        if countOfCards - (currentCardNumber + visibleCards.count) > 0 {
             
-            if visibleCards.count > 0 {
-                
+            if !visibleCards.isEmpty {
                 loadMissingCards(missingCards)
             } else {
                 setupDeck()
                 layoutDeck()
                 
-                if ((delegate?.kolodaShouldApplyAppearAnimation(self)) != false) {
+                if let delegate = delegate?.kolodaShouldApplyAppearAnimation(self) where delegate == true {
```

Comment:
Wrong naming. It would be better to call "delegate" as "shouldApply" or something like that.